src/snowflake/cli/plugins/nativeapp/codegen/snowpark/python_processor.py

```diff
@@ -140,7 +140,7 @@ def _execute_in_sandbox(
 
     if completed_process.returncode != 0:
         cc.warning(
-            f"Could not fetch Snowpark objects from {py_file} due to the following error:\n {completed_process.stderr}"
+            f"Could not fetch Snowpark objects from {py_file} due to the following Snowpark-internal error:\n {completed_process.stderr}"
```

Comment:
errors here aren't necessarily snowpark

Reply:
Errors that can go to stderr include bugs in our driver and just Python errors in user code. Labelling them all as Snowpark would be misleading.